CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 #### Fixes
 
 * your contribution
```

Comment:
Looks like we lost Danger the PR linter.

```suggestion
* Your contribution here.
```

(and it needs to be removed below in a released version)